docs/exceptions.rst

```diff
@@ -37,3 +37,9 @@ Exceptions
 
     This is raised when the verify method of a key derivation function's
     computed key does not match the expected key.
+
+
+.. class:: InvalidTag
```

Comment:
`exceptions.rst` is kind of horrible, can we document this with the rest of the symmetric ciphers stuff so someone might actually find it?